modules/text_server_adv/text_server_adv.cpp

```diff
@@ -1319,10 +1319,15 @@ _FORCE_INLINE_ bool TextServerAdvanced::_ensure_glyph(FontAdvanced *p_font_data,
 		}
 
 		if (!outline) {
+			FT_GlyphSlot slot = fd->face->glyph;
+			bool from_svg = false;
+			if (slot->format == FT_GLYPH_FORMAT_SVG) {
+				// Need to check before FT_Render_Glyph as it will change format to bitmap
+				from_svg = true;
+			}
```

Comment:
```suggestion
			bool from_svg = (slot->format == FT_GLYPH_FORMAT_SVG); // Need to check before FT_Render_Glyph as it will change format to bitmap.
```